From: Simon McVittie <smcv@debian.org>
Date: Sat, 12 Jul 2025 09:48:08 +0100
Subject: docs: Remove remotely accessed logo

Remote images in local documentation are not ideal from a privacy point
of view.

Forwarded: not-needed
---
 docs/reference/libsoup.toml.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/docs/reference/libsoup.toml.in b/docs/reference/libsoup.toml.in
index f0abab6..0236ef3 100644
--- a/docs/reference/libsoup.toml.in
+++ b/docs/reference/libsoup.toml.in
@@ -5,7 +5,6 @@ browse_url = "https://gitlab.gnome.org/GNOME/libsoup"
 repository_url = "https://gitlab.gnome.org/GNOME/libsoup.git"
 docs_url = "https://libsoup.gnome.org/libsoup-3.0/"
 website_url = "https://libsoup.gnome.org/"
-logo_url = "https://gitlab.gnome.org/uploads/-/system/project/avatar/1630/logo.png"
 authors = "Dan Winship, Claudio Saavedra, Patrick Griffis, and Carlos Garcia Campos"
 license = "LGPL-2.0-or-later"
 description = "HTTP client/server library for GNOME"
